src/DependencyInjection/SymfonyConfigCompilerPass.php

```diff
@@ -43,5 +43,9 @@ public function process(ContainerBuilder $container): void
         if (!$container->hasParameter('shopware.cache.tagging.each_theme_config')) {
             $container->setParameter('shopware.cache.tagging.each_theme_config', true);
         }
+
+        if (!$container->hasParameter('framework.secrets.enabled')) {
```

Comment:
I would check if the container has an secret command and do that based on this create a parameter